test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -1486,17 +1461,31 @@ $c.Method(
             $actual.VariablePath.UserPath | Should -Be 'provider:var'
         }
 
+        It "Fails to parse method invocation with label differing in case" {
+            $err = { ExecuteCommand '$c.Method(Arg: $value, arg: $value)' } | Should -Throw -ErrorId ParseException -PassThru
+            $pe = $err.Exception.InnerException
+            $pe | Should -BeOfType ([System.Management.Automation.ParseException])
+
+            $pe.Errors[0].ErrorId | Should -Be DuplicateArgumentLabel
+            $pe.Errors[0].Message | Should -Be 'Found duplicate argument name label "arg"'
+            $pe.ErrorRecord.InvocationInfo.PositionMessage | Should -Be (@(
```

Comment:
Should this be a `-BeExactly` to ensure the error message contained `Arg:` on the first entry?

Reply:
Yes, tbh I barely looked at the test as I just copied the one you had made for the duplicate check and changed the casing of the input.

Reply:
All good, I appreciate you fixing it up anyway!